app/views/galleries/new.html.erb

```diff
@@ -0,0 +1,23 @@
+<nav>
+  <a href="/pieces">Art Pieces</a>
+  <a href="/galleries">Galleries</a>
+</nav>
+
+<form action="/galleries" method="post">
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+
+  <p>Enter Gallery Information:</p>
+
+  <label for="gname">Gallery Name:</label>
+  <input type='text' name='gallery[name]' placeholder="Willy's Art Room" required><br><br>
+
+  <label for="gentrycost">Entry Cost (in USD):</label>
+  <input type='text' name='gallery[entry_cost]' placeholder='5.75' required><br><br>
```

Comment:
Should use a step="0.01" so that users don't enter too many decimal places of precision.

Reply:
Thank you. Since the code isn't broken and works right now, let's go ahead and merge this in. I will go back and refactor my code to use number input in my next PR.

Reply:
Agreed, we can put this change in the next refactor.